Documentation/SequentialDataStore/SDS_Streams.md

```diff
@@ -613,3 +613,46 @@ The response includes a status code.
 ```csharp
    Task UpdateStreamOwnerAsync(string streamId, Trustee streamOwner);
 ```
+*** 
+
+## `Get Stream Access Rights`
+
+Gets the AccessRights associated with the specified stream for the requesting identity. For 
```

Comment:
Any specific reason that AccessRights is one word here and two words in the next sentence?

Reply:
Just getting a little too used to coding "AccessRights" as one word, there definitely should be a space here